index.html

```diff
@@ -23,29 +23,227 @@
-                                class="fa-brands fa-angellist"></i></a></li>
-                </ul>
+                          class="fa fa-github"></i></a></li>
+                    <li><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i class="fa-brands fa-angellist"></i></a>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+          </div>
+        </div>
+    </header>
+  <main class="main">      
+    <section class="work-section">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio.svg" alt="" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>
+        <div class="links">
+          <ul class="link-items">
+            <li class="canopy">CANOPY</li>
+            <span class="list-item">•</span>
+            <li class="list-item">Back end dev</li>
+            <span class="list-item">•</span>
+            <li class="list-item">2015</li>
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class="portfolio-dashboard">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio.png" alt="Dashboard" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>
+        <div class="links">
+          <ul class="link-items">
+            <li class="canopy">CANOPY</li>
+            <span class="list-item">•</span>
+            <li class="list-item">Back end dev</li>
+            <span class="list-item">•</span>
+            <li class="list-item">2015</li>
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class="portfolio-dashboard">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio-projects.png" alt="" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>

```

Comment:
This here should use a `<hx> from <h3> to <h6>` tag. This is because it is a heading and using the correct semantic makes your site usable for all as well as friendly to screen readers